frontend/src/v5/ui/controls/breadcrumbs/breadcrumbs.styles.ts

```diff
@@ -55,6 +56,9 @@ export const InteractiveBreadcrumb = styled(Breadcrumb).attrs({
 		max-width: 100%;
 		padding-right: 9px;
 		margin-right: 0;
+		${({ secondary }) => secondary && css`
+		font-weight: ${FONT_WEIGHT.SLIM}
```

Comment:
Not a direct feedback to this code review, but do you reckon we should add a 'fontWeight' property to v5 theme so that we can be more consistent with the rest of the code and be able to write the following
```ts
${({ theme }) => `font-weight: ${theme.fontWeight.slim}`}
```

> The same goes for GRADIENT and SHADOW which must be imported otherwise

Reply:
To be honest I feel that is a bit too much for just on exception.

Reply:
If there's another case that calls for the same strategy we should add it.